app/src/main/java/com/devshish/internship/presentation/service/player/PlayerNotificationManager.kt

```diff
@@ -66,9 +69,23 @@ class PlayerNotificationManager(
     }
 
     fun onFirstPlay(song: Song?) {
+        if (song?.imageUri == null) return
```

Comment:
I'm not sure, but will notification be shown when song happens to not have Image?